storage-proofs/core/src/parameter_cache.rs

```diff
@@ -16,7 +16,7 @@ use std::io::{self, SeekFrom};
 use std::path::{Path, PathBuf};
 
 /// Bump this when circuits change to invalidate the cache.
-pub const VERSION: usize = 27;
+pub const VERSION: usize = 28;
```

Comment:
Oh, this requires a version bump here to 28?

Reply:
this version shouldn’t change, only when we have published 27 and need to change sth

Reply:
Hmmmm. I guess we have concretized a more recent policy. I'll go with that but note it means developers will more often get caught with stale parameters.